src/lib/Menu/ContentRightSidebarBuilder.php

```diff
@@ -443,6 +554,63 @@ private function getCopySubtreeLimit(): int
             'subtree_operations.copy_subtree.limit'
         );
     }
+
+    /**
+     * @param \Ibexa\Contracts\Core\Repository\Values\Content\Location|null $location
+     * @param \Ibexa\Contracts\Core\Repository\Values\Content\Content $content
+     * @param \Ibexa\Contracts\Core\Repository\Values\Content\Language $language
+     * @param array $options
```

Comment:
that could be probably typehinted at least to <string, mixed>  to drop that phpstan warning.